openslides_backend/presenter/get_user_related_models.py

```diff
@@ -90,22 +89,41 @@ def get_committees_data(self, user: dict[str, Any]) -> list[dict[str, Any]]:
         return committees_data
 
     def get_meetings_data(self, user: dict[str, Any]) -> list[dict[str, Any]]:
-        if not user.get("meeting_user_ids"):
+        if not (meeting_user_ids := user.get("meeting_user_ids")):
             return []
 
         result_fields = (
             "speaker_ids",
             "motion_submitter_ids",
             "assignment_candidate_ids",
         )
-        meeting_users = self.datastore.filter(
-            "meeting_user",
-            And(
-                FilterOperator("user_id", "=", user["id"]),
-                FilterOperator("group_ids", "!=", []),
-            ),
-            ["meeting_id", *result_fields],
-        ).values()
+        meeting_users = list(
+            self.datastore.get_many(
+                [
+                    GetManyRequest(
+                        "meeting_user",
+                        meeting_user_ids,
+                        [*result_fields, "group_ids", "meeting_id"],
+                    )
+                ]
+            )["meeting_user"].values()
+        )
+        meeting_users = [
```

Comment:
This part i snot necessary. After line 110 the meeeting_users are read in in memory. Something like
`meeting_user = [meeting_user for meeting_user in meeting_users if meeting_user.pop("group_ids", None)]`
is sufficient.
Then continue with 125

Reply:
Deleted the other part